grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -2662,6 +2666,307 @@
         "alignLevel": null
       }
     },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "shows how much space the data stored in the clickhouse takes compared to all the free space inside the kubernetes pod\n\nbe careful with multiple filesystem volumes, kubernetes volume claims and S3 as storage backend ",
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 8,
+        "w": 8,
+        "x": 0,
+        "y": 53
+      },
+      "hiddenSeries": false,
+      "id": 39,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [
+        {
+          "targetBlank": true,
+          "title": "filesystemFree()",
+          "url": "https://clickhouse.tech/docs/en/sql-reference/functions/other-functions/#filesystemfree"
+        },
+        {
+          "targetBlank": true,
+          "title": "system.parts",
+          "url": "https://clickhouse.tech/docs/en/operations/system-tables/#system_tables-parts"
+        },
+        {
+          "targetBlank": true,
+          "title": "Multiple Disk Volumes",
+          "url": "https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/mergetree/#table_engine-mergetree-multiple-volumes"
+        }
+      ],
+      "nullPointMode": "null",
+      "options": {
+        "dataLinks": []
+      },
+      "percentage": false,
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "chi_clickhouse_metric_DiskFreeBytes{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"} / (chi_clickhouse_metric_DiskFreeBytes{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"} + chi_clickhouse_metric_DiskDataBytes{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"})",
```

Comment:
Seems wrong

Reply:
the current implementation looks like that:
![image](https://user-images.githubusercontent.com/105560/82557525-53605800-9b85-11ea-8d03-df5f5611cf69.png)

do you prefer just show DiskFreeBytes? instead of percent?

Reply:
100% looks bad on a graph. Let's change it to 'Disk Used', so it would be close to zero on empty system.
Disk Used = Data / (Free + Data)

Reply:
@alex-zaitsev fixed on https://github.com/Altinity/clickhouse-operator/pull/391/commits/e359103735cdea626e8a54f7f0b6488874a8aa0c

current look and feel
![image](https://user-images.githubusercontent.com/105560/82656831-d3e28f80-9c3d-11ea-9ee7-c5126b8e91b5.png)